logistics/scripts/modules/TestHarness.psm1

```diff
@@ -61,6 +61,20 @@ function Get-TestHarnessExecutable {
     return $testHarnessExecutable
 }
 
+function Invoke-SdkGenConsole {
+    Param(
+        [Parameter(Mandatory = $true)] [string] $apiUrl, 
+        [string] $buildConfiguration = "Debug"
+    )
+    
+    $sdkGenConsoleFolder = (Get-RepositoryResolvedPath "\Utilities\SdkGen\EdFi.SdkGen.Console")
+    $sdkGenConsoleExeName = (Get-Item $sdkGenConsoleFolder).Name
+    $sdkGenConsoleExecutableFolder = "$sdkGenConsoleFolder\bin\$buildConfiguration\netcoreapp3.1"
+    $params = "/c $sdkGenConsoleExeName.exe -m "+ $apiUrl+"/metadata?sdk=true -v 2.4.15 --include-profiles --include-composites --include-identity"
+    Write-Host "$sdkGenConsoleExecutableFolder $params" -ForegroundColor Magenta
+    Start-Process -FilePath cmd -WorkingDirectory $sdkGenConsoleExecutableFolder -verb runas -ArgumentList ($params) -PassThru -Wait
```

Comment:
we should be using & $params to run this build.

Reply:
https://intedfitools1.msdf.org/buildConfiguration/OdsPlatform_OdsApi_Packages_EdFiOdsApiTestSdk/128693